README.md

```diff
@@ -40,27 +38,30 @@ The IDE is just a simple node project with few dependencies, and can be setup wi
 git clone --recursive https://github.com/thoth-tech/SplashkitOnline.git
 cd SplashkitOnline/Browser_IDE/
 npm install
+```
+However, we also need to import the SplashKit library, as its not included in the repository by default. The Node server will import the necessary files automatically on start-up, but this can also be achieved with the included `setup.py` script.
+```bash
+py setup.py # optional
```

Comment:
This should probably be changed to `python3 setup.py` as `py` is not present on all OS's -- For example, this command does not work in WSL2.

It could also be explained that this command should be run from the root of the project, as the following command `npm run server` must be run from the Browser_IDE folder.

Reply:
Thank you for the note. I was unsure which tool name was the canonical one.